public/tokenMap.json

```diff
@@ -10080,11 +10080,6 @@
       "decimals": 18,
       "symbol": "CAI"
     },
-    "zora": {
-      "address": "0xD8E3FB3b08eBA982F2754988d70D57eDc0055ae6",
-      "decimals": 9,
-      "symbol": "ZORA"
-    },
```

Comment:
This token looks perfectly fine... 

https://etherscan.io/token/0xD8E3FB3b08eBA982F2754988d70D57eDc0055ae6